openstudio-building-energy-standard-measures.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |spec|
     spec.add_development_dependency 'rubocop', '~> 1.50'
```

Comment:
Line 26 - if OSSTD 0.2.12 is no longer supported then we should just reject the use of this measure with lower ruby version.